app/src/main/java/com/coderGtm/yantra/Utils.kt

```diff
@@ -102,8 +102,9 @@ fun contactsManager(terminal: Terminal, callingIntent: Boolean = false, callTo:
                             cursorPhone.getColumnIndex(ContactsContract.CommonDataKinds.Phone.NUMBER))
                         builder.add(Contacts(name,phoneNumValue))
                         terminal.contactNames.add(name)
-                        if (callingIntent && callTo == name.lowercase() && !callingCandidates.contains(phoneNumValue)) {
-                            callingCandidates.add(phoneNumValue)
+                        val phoneNumValueStandardized  = phoneNumValue.filter { !it.isWhitespace() }
```

Comment:
Instead of putting the negation in closure, using filterNot function is more syntatic and clear

Reply:
thanks for the feedback. I will address that

Reply:
Done